### PR TITLE
[bilibili] Update payload

### PR DESCRIPTION
**Pull request by ctgnauh**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [x] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Support full HD quality.

### CODE REVIEW

File: youtube_dl/extractor/bilibili.py [hunk 1]
```diff
@@ -113,7 +113,7 @@ def _real_extract(self, url):
                 self._report_error(js)
             cid = js['result']['cid']
 
-        payload = 'appkey=%s&cid=%s&otype=json&quality=2&type=mp4' % (self._APP_KEY, cid)
+        payload = 'appkey=%s&cid=%s&otype=json&quality=0&type=flv' % (self._APP_KEY, cid)
```
Comment (dstftw) [inline youtube_dl/extractor/bilibili.py]: 1. Both should be extracted.
2. Update tests.

Comment (remitamine) [reply youtube_dl/extractor/bilibili.py]: > 1. Both should be extracted.

https://github.com/rg3/youtube-dl/issues/12095#issuecomment-280003212

Comment (ctgnauh) [reply youtube_dl/extractor/bilibili.py]: 2. Tests have been updated.

**Comment by stardiviner** [comment]
Hi, any updates on here. I'm integrating youtube-dl to other Emacs tools, hope this improvement can be merged.

**Comment by yan12125** [comment]
@stardiviner: the issue mentioned by @remitamine should be fixed first

**Pull request closed without merging**